### DISCUSSION TITLE
SSL certificate still expired after Update

### DISCUSSION THREAD

**Discussion by modem-19** [amidaware/tacticalrmm#1884] [General]
I'm still in a testing phase of Tactical RMM and while testing, I forgot to manually renew my cert with lets-encrypt.  So I went through, backed up my DB, updated to 0.18.2, ran the cert upgrade command, and finally rerunning the update.sh --force command.  Unfortunately, on my RMM main page, it's still showing that "SSL certificate expires in 0 days".

I suspect I didn't renew it before it completely expired.  What are the steps to rectify this?

Btw, I love this product, and even a security engineer at the MSP I work gave high praises to this over ConnectWise RMM for more commonsense functions found in RMM and Mesh.

**Comment by wh1te909** [comment]
Please use the `history` command to find and copy/paste the exact certbot command you ran to renew the cert. Replace the domain and email with `example.com`
Also paste output of `ls -l /etc/letsencrypt/live/`

**Reply by modem-19** [reply]
@wh1te909 Here's the requested info:

`sudo certbot certonly --manual -d *.rmm.EXAMPLE.COM --agree-tos --no-bootstrap --preferred challenges dns -m alerts@EXAMPLE.COM -no-eff-email`

Then for the command output:

`total 24
drwxr-xr-x 2 tactical tactical 4096 Mar 12 01:15 api.EXAMPLE.COM
drwxr-xr-x 2 tactical tactical 4096 Mar 11 05:20 mesh.EXAMPLE.COM
drwxr-xr-x 2 tactical tactical 4096 Mar 12 01:45 EXAMPLE.COM
-rw-r--r-- 1 tactical tactical 740 Dec 15 03:11 README
drwxr-xr-x 2 tactical tactical 4096 Mar 11 05:10 rmm.EXAMPLE.COM
drwxr-xr-x 2 tactical tactical 4096 Jun  9 02:09 rmm.EXAMPLE.COM-0001`

**Reply by wh1te909** [reply]
you need to get a wildcard cert, not a cert for `rmm.example.com` so in your certbot command do `*.example.com` NOT `*.rmm.example.com`

**Reply by modem-19** [reply]
@wh1te909 thanks for the assistance, worked like a charm!  I was thinking because I had it as a subdomain that I had to use that.  I made a note for doing that again next time.
